frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -147,11 +150,24 @@ const flavorUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) =>
 
 const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
   const state = getState();
-  if (!hasVMSettingsValueChanged(prevState, state, id, VMSettingsField.OPERATING_SYSTEM)) {
+  if (
+    !hasVMSettingsValueChanged(
```

Comment:
windowsTools do not depend on these. Can we split the golden image logic from this updater?

Reply:
splited